unattended-upgrade

```diff
@@ -336,22 +336,22 @@ class LogInstallProgress(apt.progress.base.InstallProgress):
 
 class Unlocked:
     """
-    Context manager for unlocking the apt lock while cache.commit() is run
+    Context manager for unlocking the apt lock for cache.commit() or dpkg call
     """
 
     def __enter__(self):
         # type: () -> None
         try:
             apt_pkg.pkgsystem_unlock()
         except Exception:
-            pass
+            raise
```

Comment:
@mvo5 This is why we missed locking problems earlier. :-(

Reply:
Thanks for finding this!

Reply:
Given that try/except now just raises, we might as well remove it, no?

Reply:
True, fixed.
Originally I wanted to add some logging but later decided it would have been redundant.